README.md

```diff
@@ -0,0 +1,78 @@
+# Moltimate Frontend
+
+Moltimate is an open source tool built to aid in protein function analysis. The project start with a group of student developers in the Software Engineering Department at Rochester Institute of Technology. Main dependencies for this project are  [React](https://reactjs.org/docs/getting-started.html) , [Material UI](https://material-ui.com/), [NGL View](https://github.com/arose/nglview).
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Must have Node version greater than 10 
+```
+node -v
+```
+If not installed then use the below command
+```
+brew install node
```

Comment:
Can you reference Node's documentation instead of giving a Mac-specific command? https://nodejs.org/en/download/ & https://nodejs.org/en/download/package-manager/